index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>mint.com signup page</title>
+</head>
+<body>
+    <!-- Logo -->
+    <header>
+        <div class="header">
+            <img src="images/logo.png" alt="Intuit Logo">
```

Comment:
- [ ] Kindly add all the other items on the top of the page. When you put all the items, do not use screenshots, instead, type the words, and only add the logos.

Source | Clone
-- | --
![image](https://user-images.githubusercontent.com/57414789/83826333-45d8c100-a6a1-11ea-8e9a-4ac526f3affa.png) | ![image](https://user-images.githubusercontent.com/57414789/83826314-378aa500-a6a1-11ea-8d1c-440667938831.png)